src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapSettings.jsx

```diff
@@ -5,31 +5,31 @@ import {
 import {
   Button, Dropdown, Menu, Tooltip,
 } from 'antd';
+import PropTypes from 'prop-types';
 import HeatmapMetadataTrackSettings from './HeatmapMetadataTrackSettings';
 import HeatmapGroupBySettings from './HeatmapGroupBySettings';
+import HeatmapExpressionValuesSettings from './HeatmapExpressionValuesSettings';
 import HeatmapLegendVisibilitySettings from './HeatmapLegendVisibilitySettings';
 
 const { SubMenu, Item } = Menu;
 
-const HeatmapSettings = () => {
+const HeatmapSettings = (props) => {
```

Comment:
you don't seem to use any `props` in here - if that is the case you can remove this `props` in here

Reply:
Got carried over, like the last comment. Deleted in commit c5eba06.